doc/release-notes/1.2.0.txt

```diff
@@ -4,7 +4,25 @@ TeleSculptor v1.2.0 Release Notes
 This is a minor release of TeleSculptor that provides both new functionality
 and fixes over the previous v1.1.0 release.
 
-Updates since v1.1.0
+The biggest changes in this release are the transition to a new Sphinx-based
+user manual at https://telesculptor.readthedocs.io, the removal of deprecated
+MAP-Tk command line tools, improved handling of mesh coloring from the video,
+and an interface for manual camera calibration by manually placing camera
+registration points on frames.
+
+TeleSculptor v1.2.0 also builds on updated KWIVER v1.6.0 and Fletch v1.5.0,
```

Comment:
This is of course technically not true since it a commit tag past the 1.6.0 release.

Reply:
close enough, we don't have 1.6.1 out yet